packages/sdk-ts/src/client/indexer/transformers/IndexerGrpcExplorerTransformer.ts

```diff
@@ -1,3 +1,5 @@
+import { BigNumberInWei, BigNumberInBase } from '@injectivelabs/utils'
+import { isJsonString } from '../../../utils/helpers.js'
```

Comment:
can consider to sort the import as our pattern
```suggestion
import { InjectiveExplorerRpc } from '@injectivelabs/indexer-proto-ts'
import { BigNumberInWei, BigNumberInBase } from '@injectivelabs/utils'
import { isJsonString } from '../../../utils/helpers.js'
import { grpcPagingToPaging } from '../../../utils/index.js'
import {
  Block,
  GasFee,
  Message,
  ExplorerTransaction,
  ContractTransaction,
  GrpcValidatorSlashingEvent,
  ExplorerValidatorDescription,
  GrpcIndexerValidatorDescription
} from '../types/explorer.js'
```